src/main/views/claim-saved.njk

```diff
@@ -12,4 +13,9 @@
         {{ govukPanel({
         titleText: "Your claim has been saved"
       }) }}
+      {{ govukButton({
+        text: continueWithClaim,
```

Comment:
missing claim-saved.json translation file. You added continueWithClaim entry to home.json but do not reference home.json in the YourClaimHasBeenSavedController controller. I think you need to create translation file claim-saved.json and move the continueWithClaim field to it.